forgettable-frontend/src/pages/edit/EditPerson.js

```diff
@@ -214,7 +215,7 @@ export default function EditPerson() {
         <div className={classes.row}>
           <div className={classes.column}>
             <InputField inputID='first_name' placeholder={'E.g. John'} inputLabel='First Name' inputType='primary' inputStateValue={personData.first_name} autoFocusState={true} requiredState/>
-            <InputField inputID='gender' placeholder={'E.g. Female'} inputLabel='Gender' inputType='primary' inputStateValue={personData.gender}/>
+            <InputSelector inputID='gender' placeholder={'E.g. Female'} inputLabel='Gender' inputStateValue={personData.gender} options={['Male', 'Female', 'Prefer not to say', 'Unlisted']}/>
```

Comment:
Another pedantic remark mainly because I can't find anything detrimental, but we could separate out these as a constant array. Although gender doesn't change, it is a better practice not to inline options like that.

Reply:
good point, fixed